app/docs/user.md

```diff
@@ -1,3 +1,195 @@
-# YourFunctionName
+# buckinghamPiTheorem
```

Comment:
See comments on the dev docs, which mostly apply to this file due to the duplicaiton.

Is the duplication necessary? Could the dev docs point to the user docs and then just have implementation-related details?

Reply:
Dev docs now point to user docs